src/v2/cookbook/debugging-in-vscode.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Debugging in VS Code and Chrome
+type: cookbook
+order: 6
+---
+
+Every application reaches a point where it's necessary to understand failures, small to large. In this recipe, we explore a few workflows for VS Code users, who are using Chrome to test.
+
+This recipe shows how to use the [Debugger for Chrome](https://github.com/Microsoft/VSCode-chrome-debug) extension with VS Code to debug Vue.js applications generated by the [Vue CLI](https://github.com/vuejs/vue-cli).
+
+## Prerequisites
+
+You must have Chrome and VS Code installed. Make sure to the latest version of [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome) extension installed in VS Code.
+
+Install and create a project with the [vue-cli](https://github.com/vuejs/vue-cli) by running
+
+```
+npm install -g @vue/cli
+# or
+yarn global add @vue/cli
+
+vue create my-project
+```
+
+Change to the newly created application directory and open VS Code.
+
+```
+cd my-project
+
+# if you have an alias set up (recommended)
+code .
+```
+
+### Update your webpack configuration
+
+Before you can debug your Vue components from VS Code you need to update the generated webpack config to build sourcemaps that contains more information for our debugger.
+
+* Go to `config/index.js` and find the `devtool` property. Update it to:
+
+```json
+devtool: 'source-map',
+```
+
+### Configure launch.json File
```

Comment:
`launch.json` should probably be in backticks.

Reply:
Actually, with these titles, I wonder if it might be better to use titles that describe what we're trying to achieve, instead of the implementation details to get there.

For example, instead of `Update your webpack configuration`, we could say `Showing Source Code in the Chrome Devtools`. Then instead of `Configure launch.json File`, we could say, `Launching Our App from VS Code`.

Reply:
Yeah, that makes more sense and is probably more easily googleable (not a word, but I'm keeping it)